README.md

```diff
@@ -1,10 +1,44 @@
-## rapids-js
+# <div align="left"><img src="https://rapids.ai/assets/images/rapids_logo.png" width="90px"/>&nbsp; rapids-js
 
-### Setup
 
-To get started with `rapids-js`, follow the [Setup Instructions](docs/setup.md).
+The rapids-js project comprises of collection of js bindings to the following Rapids libraries:
+1. [cudf](https://github.com/rapidsai/cudf)
+2. [cugraph](https://github.com/rapidsai/cugraph)
+3. [cuspatial](https://github.com/rapidsai/cuspatial)
+4. [rmm](https://github.com/rapidsai/rmm)
 
-### License
+Apart from the Rapids libraries, this project also includes js bindings to the following libraries:
+1. cuda
+2. [glfw](https://github.com/glfw/glfw)
+3. webgl
```

Comment:
```suggestion
Additionally, `rapids-js` includes JavaScript bindings to [cuda](https://developer.nvidia.com/cuda-toolkit) and [glfw](https://github.com/glfw/glfw).
```

(Left off webgl since webgl is already JS to being with?)